bin/resources/GameIndex.yaml

```diff
@@ -32247,7 +32247,7 @@ SLPM-55143:
 SLPM-55144:
   name: "NBA LIVE 08 [EA BEST HITS]"
   name-sort: "NBA らいぶ08 [EA BEST HITS]"
-  name-en: "NBA Live 08 [EA BEST HITS]"
+  name-en: "NBA Live 08 [EA Best Hits]"
```

Comment:
any reason your changing this ?

Reply:
The majority of tags written in English are not written in uppercase letters, so we simply unified them to match that.

There is no deeper meaning than that.